### PR TITLE
CLI: increase coverage of the "cli" module

### PR DESCRIPTION
**Pull request by markelog**

### CODE REVIEW

File: test/specs/cli.js [hunk 1]
```diff
@@ -85,6 +87,52 @@ describe('modules/cli', function() {
...
+            args: ['test/data/cli/error.js'],
+            preset: 'google'
+        });
+
+        return result.promise.always(function() {
+            var i;
+            var config = Checker.prototype.configure.getCall(0).args[0];
+
+            for (i in config) {
+                assert(false);
```
Comment (mrjoelkemp) [inline test/specs/cli.js]: assert(!Object.keys(config)) instead of the for and assert(true)?

Comment (markelog) [inline test/specs/cli.js]: Cool, updated

**Comment by mrjoelkemp** [comment]
Lgtm

**Comment by zxqfox** [comment]
:+1: Same

Can we land this?

**Pull request closed without merging**